### PR TITLE
Add section on Module Unification Packages

### PR DESCRIPTION
**Pull request by Alonski**
This might be a bit too long? Not sure where to cut it or if I maybe need to add more?

### CODE REVIEW

File: source/blog/2018-08-31-the-ember-times-issue-62.md [hunk 1]
```diff
@@ -41,8 +41,47 @@ Read more in the [rendered pull request on GitHub](https://github.com/emberjs/rf
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Module Unification with Ember Addons](https://github.com/emberjs/rfcs/pull/367) 🎁
+[Module Unification Packages](https://github.com/emberjs/rfcs/pull/367) is a new RFC, created by [@mixonic](https://github.com/mixonic), that sets out to describe how packages, **Ember Addons**, work and how apps and addons will migrate to the new [Module Unification](https://github.com/emberjs/rfcs/blob/master/text/0143-module-unification.md) structure from the classic structure. This RFC iterates on and is set to replace another RFC called [Module Unification Namespaces](https://github.com/emberjs/rfcs/pull/309) which had some syntax, like the `::` syntax, that proved problematic.
```
Comment (mixonic) [inline source/blog/2018-08-31-the-ember-times-issue-62.md]: > that sets out to describe how packages, **Ember Addons**

I suggest sticking with "describe how Ember apps and addons will migrate to the new...". The terminology of "package" is confusing if this is someones first introduction to the RFC. Focusing on "this is how addons and apps work under MU" is a good high level message.

File: source/blog/2018-08-31-the-ember-times-issue-62.md [hunk 2]
```diff
@@ -41,8 +41,47 @@ Read more in the [rendered pull request on GitHub](https://github.com/emberjs/rf
...
+
+This RFC proposes to add a new `{{use}}` helper that will be used to call external components that are provided by addons. This helper is a subset of the javascript imports that we are used to. 
+
+An example: In this template the `{{use}}` helper imports a component `Widget` from the `gadget` addon.
+
+```hbs
+{{! invokes node_modules/gadget/src/ui/components/Widget/component.js }}
+
+{{use Widget from 'gadget'}}
+<Widget @options=someOptions @value=someValue />
```
Comment (mixonic) [inline source/blog/2018-08-31-the-ember-times-issue-62.md]: My syntax here was incorrect, I need to fix it in the RFC. This should be:

```hbs
{{! invokes node_modules/gadget/src/ui/components/Widget/component.js }}

{{use Widget from 'gadget'}}
<Widget @options={{someOptions}} @value={{someValue}} />
```

File: source/blog/2018-08-31-the-ember-times-issue-62.md [hunk 3]
```diff
@@ -41,8 +41,47 @@ Read more in the [rendered pull request on GitHub](https://github.com/emberjs/rf
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Module Unification with Ember Addons](https://github.com/emberjs/rfcs/pull/367) 🎁
+[Module Unification Packages](https://github.com/emberjs/rfcs/pull/367) is a new RFC, created by [@mixonic](https://github.com/mixonic), that sets out to describe how packages, **Ember Addons**, work and how apps and addons will migrate to the new [Module Unification](https://github.com/emberjs/rfcs/blob/master/text/0143-module-unification.md) structure from the classic structure. This RFC iterates on and is set to replace another RFC called [Module Unification Namespaces](https://github.com/emberjs/rfcs/pull/309) which had some syntax, like the `::` syntax, that proved problematic.
+
+This RFC proposes to add a new `{{use}}` helper that will be used to call external components that are provided by addons. This helper is a subset of the javascript imports that we are used to.
```
Comment (mixonic) [inline source/blog/2018-08-31-the-ember-times-issue-62.md]: Maybe:

> This RFC proposes to add a new `{{use}}` helper. This helper imports components from an addon into an application's template. The helper provides a subset of the functionality JavaScript imports have, but it syntactically slightly different.

File: source/blog/2018-08-31-the-ember-times-issue-62.md (hunk 2 above)
Comment (Alonski) [reply source/blog/2018-08-31-the-ember-times-issue-62.md]: Oh good catch!

File: source/blog/2018-08-31-the-ember-times-issue-62.md (hunk 1 above)
Comment (Alonski) [reply source/blog/2018-08-31-the-ember-times-issue-62.md]: Ok makes sense. Thanks

File: source/blog/2018-08-31-the-ember-times-issue-62.md [hunk 4]
```diff
@@ -41,8 +41,47 @@ Read more in the [rendered pull request on GitHub](https://github.com/emberjs/rf
...
+  // inject node_modules/ember-stripe-service/src/services/store.js
+  checkoutService: inject('stripe', { package: 'ember-stripe-service' }),
+
+  // inject node_modules/ember-simple-auth/src/services/session.js
+  session: inject({ package: 'ember-simple-auth' })
+
+});
+```
+
+There is also some proposals regarding `owner` APIs such as `owner.lookup()` and `owner.factoryFor()` which have also become more explicit.
```
Comment (mixonic) [inline source/blog/2018-08-31-the-ember-times-issue-62.md]: > There are also some proposals

File: source/blog/2018-08-31-the-ember-times-issue-62.md [hunk 5]
```diff
@@ -41,8 +41,47 @@ Read more in the [rendered pull request on GitHub](https://github.com/emberjs/rf
...
+
+All in all this, **very well written**, RFC is a great chance to learn about possible changes to Ember and the Module Unification structure.
+If you have any concerns or questions feel free to ask in the [RFC issue](https://github.com/emberjs/rfcs/pull/367) and join in on the conversation.
```
Comment (mixonic) [inline source/blog/2018-08-31-the-ember-times-issue-62.md]: > join in the conversation by joining `#st-module-unification` on Ember.js Community Slack.

**Comment by Alonski** [comment]
Finished fixing comments by @mixonic. Thanks for the help!

**Pull request merged**